### ISSUE TITLE
All files seems to be broken or empty

### ISSUE THREAD

**Issue by michalastocki** [renanxcortes/springerQuarantineBooksR#53]
Does downloading work for you right now?

For me all the files are very small, the same size, and don't open at all.

![image](https://user-images.githubusercontent.com/51298466/82102609-5d5f0280-9710-11ea-9c09-41fbd6a4962e.png)

**Comment by renanxcortes** [comment]
Yes, I'm checking this issue now... It is happening on my side also. Thanks for pointing tat out!

**Comment by renanxcortes** [comment]
I have that this issue is also happening in the Python version of the library: https://github.com/alexgand/springer_free_books/issues/113
It seems like Springer added an additional security step with some kind of recaptcha.

**Comment by mutantacule** [comment]
hey, 
just add the cookie `recaptcha=` to your request to make it work again

**Comment by renanxcortes** [comment]
> hey,
> just add the cookie `recaptcha=` to your request to make it work again

That's really cool! I still need to investigate how to adapt your Python solution in R. Do you have any clue that helps me in that sense? Thanks!

**Comment by renanxcortes** [comment]
> > hey,
> > just add the cookie `recaptcha=` to your request to make it work again
> 
> That's really cool! I still need to investigate how to adapt your Python solution in R. Do you have any clue that helps me in that sense? Thanks!

(Perhaps an additional argument in https://github.com/renanxcortes/springerQuarantineBooksR/blob/e0b76b697225674171046f8e4c3a209dfeffb8b4/R/download_springer_book.R#L50)

**Comment by mutantacule** [comment]
> > > hey,
> > > just add the cookie `recaptcha=` to your request to make it work again
> > 
> > 
> > That's really cool! I still need to investigate how to adapt your Python solution in R. Do you have any clue that helps me in that sense? Thanks!
> 
> (Perhaps an additional argument in
> 
> https://github.com/renanxcortes/springerQuarantineBooksR/blob/e0b76b697225674171046f8e4c3a209dfeffb8b4/R/download_springer_book.R#L50
> 
> )

Hey! Thanks for pointing to this file. I'm not exactly familiar with R, but from what I found, this should do it:

```
GET(download_url, set_cookies(`recaptcha` = "")) 
```

Let me know if it worked!

**Comment by renanxcortes** [comment]
> > > > hey,
> > > > just add the cookie `recaptcha=` to your request to make it work again
> > > 
> > > 
> > > That's really cool! I still need to investigate how to adapt your Python solution in R. Do you have any clue that helps me in that sense? Thanks!
> > 
> > 
> > (Perhaps an additional argument in
> > https://github.com/renanxcortes/springerQuarantineBooksR/blob/e0b76b697225674171046f8e4c3a209dfeffb8b4/R/download_springer_book.R#L50
> > 
> > )
> 
> Hey! Thanks for pointing to this file. I'm not exactly familiar with R, but from what I found, this should do it:
> 
> ```
> GET(download_url, set_cookies(`recaptcha` = "")) 
> ```
> 
> Let me know if it worked!

Awesome! I made many tests locally here and it worked, thank you so much, @mutantacule ! I'll fix that through a PR!

**Comment by renanxcortes** [comment]
This issue is now reopened. It seems like springer tweaked the reCaptcha and the workaround is not working anymore :(

**Comment by Raymo111** [comment]
@renanxcortes try randomizing IP.
